keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -99,48 +101,89 @@ Other identified non-goals are:
 ### API Changes
 
 PodSpec becomes mutable with regards to Container resources requests and
-limits. PodSpec is extended with information of resources allocated on the
-Node for the Pod. PodStatus is extended to show the actual resources applied
+limits. PodStatus is extended to show the resources allocated for and applied
 to the Pod and its Containers.
 
 Thanks to the above:
 * Pod.Spec.Containers[i].Resources becomes purely a declaration, denoting the
   **desired** state of Pod resources,
-* Pod.Spec.Containers[i].ResourcesAllocated (new object, type v1.ResourceList)
-  denotes the Node resources **allocated** to the Pod and its Containers,
-* Pod.Status.ContainerStatuses[i].Resources (new object, type
+* Pod.Status.ContainerStatuses[i].ResourcesAllocated (new field, type
+  v1.ResourceList) denotes the Node resources **allocated** to the Pod and its
+  Containers,
+* Pod.Status.ContainerStatuses[i].Resources (new field, type
   v1.ResourceRequirements) shows the **actual** resources held by the Pod and
   its Containers.
+* Pod.Status.ContainerStatuses[i].Resize (new field, type map[string]string)
```

Comment:
After sleeping over it, I'm still leaning towards keeping this at Pod level. Today, we either admit a pod giving it the entire resource ask, or we don't. I think we should continue to hold that same bar for resizes even if a single container is being resized.

Reply:
ACK - new commit pushed.